source/grid/grid_tools_cache.cc

```diff
@@ -146,6 +152,38 @@ namespace GridTools
       }
     return vertex_kdtree;
   }
+
+
+
+  template <int dim, int spacedim>
+  const RTree<std::pair<BoundingBox<spacedim>, unsigned int>> &
+  Cache<dim, spacedim>::get_covering_rtree() const
+  {
+    if (update_flags & update_covering_rtree)
+      {
+        // TO DO: the locally owned portion of the domain
+        // might consists of two separate pieces: a single
+        // bounding box might not always be the best choice.
+        IteratorFilters::LocallyOwnedCell locally_owned_cell_predicate;
+        const BoundingBox<spacedim>       bbox(
+          GridTools::compute_bounding_box(*tria, locally_owned_cell_predicate));
```

Comment:
not wrong, but easier to read if you write it as
```
  Type var = initializer;
```

Reply:
Sorry, I can't understand what change you're suggesting...

Reply:
```
const BoundingBox<spacedim> bbox = GridTools::compute_bounding_box(*tria, locally_owned_cell_predicate);
```